### ISSUE TITLE
[BUG]: Async-version: the cache-directory and files are not created.

### ISSUE THREAD

**Issue by bouwew** [plugwise/plugwise_usb-beta#174]
### Describe the bug.

When I add the Plugwise USB beta integration as a new Integration, it fails with an error about the cache-directory not being present.

### What version of Plugwise Beta are you using?

v0.50.0a4

### What was the last working version of Plugwise Beta (or Core if not using beta before)?

v0.40.7

### What version of Home Assistant Core are you using?

core-2024.8.2

### What type of installation are you running?

Home Assistant Supervised

### How did you install plugwise-beta?

HACS

### What kind of Plugwise device is having issues?

Stick (USB)

### What firmware version is your Plugwise product at?

Unknown

### Logging

```
Logger: homeassistant.config_entries
Source: config_entries.py:604
First occurred: 16:43:30 (1 occurrences)
Last logged: 16:43:30

Error setting up entry Stick for plugwise_usb
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 604, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/plugwise_usb/__init__.py", line 82, in async_setup_entry
    await api_stick.discover_coordinator(load=False)
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/__init__.py", line 312, in discover_coordinator
    await self._network.discover_network_coordinator(load=load)
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/network/__init__.py", line 269, in discover_network_coordinator
    if await self._discover_node(
       ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/network/__init__.py", line 429, in _discover_node
    await self._notify_node_event_subscribers(NodeEvent.DISCOVERED, mac)
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/network/__init__.py", line 575, in _notify_node_event_subscribers
    await gather(*callback_list)
  File "/config/custom_components/plugwise_usb/__init__.py", line 71, in async_node_discovered
    await node.load()
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/nodes/circle_plus.py", line 37, in load
    if await self._load_from_cache():
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/nodes/circle.py", line 745, in _load_from_cache
    if not await super()._load_from_cache():
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/nodes/__init__.py", line 404, in _load_from_cache
    if not await self._load_cache_file():
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/nodes/__init__.py", line 392, in _load_cache_file
    await self._node_cache.initialize_cache()
  File "/usr/local/lib/python3.12/site-packages/plugwise_usb/helpers/cache.py", line 52, in initialize_cache
    raise CacheError(f"Unable to initialize caching. Cache folder '{self._root_dir}' does not exists.")
plugwise_usb.exceptions.CacheError: Unable to initialize caching. Cache folder '/config/.storage/plugwisecache-01J62DF8NMF7C4W0ZCQAXDA2Z8' does not exists.
```

### Additional information

My HA system is running in a Proxmox virtual container.

**Comment by brefra** [comment]
Should be fixed in v0.50.0a7

**Comment by bouwew** [comment]
Thanks! I'll test and report back.

**Comment by bouwew** [comment]
After adding the integration again I see it has found the Stealth M+.
But now I see this in HA Logs:
```
This error originated from a custom integration.

Logger: plugwise_usb.helpers.cache
Source: custom_components/plugwise_usb/__init__.py:71
integration: Plugwise USB Beta (documentation)
First occurred: 09:04:30 (7 occurrences)
Last logged: 09:04:48

OS error [Errno 2] No such file or directory: '/config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0004A1D580.cache' while reading cache file /config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0004A1D580.cache
OS error [Errno 2] No such file or directory: '/config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0003562D3C.cache' while reading cache file /config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0003562D3C.cache
OS error [Errno 2] No such file or directory: '/config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0003562C9E.cache' while reading cache file /config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0003562C9E.cache
OS error [Errno 2] No such file or directory: '/config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0003562CC1.cache' while reading cache file /config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0003562CC1.cache
OS error [Errno 2] No such file or directory: '/config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0003562003.cache' while reading cache file /config/.storage/plugwisecache-01J645KGX3XPKCBQR6RT25F2FR/000D6F0003562003.cache
```

**Comment by bouwew** [comment]
And I see these messages twice:
![image](https://github.com/user-attachments/assets/2ac88983-77f8-4d1f-99d9-d4b35eaee50d)

**Comment by bouwew** [comment]
I'm now seeing all 7 Circles too.

And I'm seeing this in .storage:
![image](https://github.com/user-attachments/assets/7c4b58e0-0ea2-41b8-a52a-3af0aea1cabb)

**Comment by bouwew** [comment]
Tested to be solved in v0.50.0a9, thanks! @brefra :)
